src/App.vue

```diff
@@ -2,7 +2,7 @@
   <div id="app">
     <TopNav/>
     <div id="content">
-      <router-view/>
+      <router-view :key="$route.fullPath"/>
```

Comment:
Well done you found the trick! Seem to properly solve the page loading old issue :)